core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.component.ts

```diff
@@ -105,7 +105,7 @@ export class SkillsListComponent {
           }, 100);
         }
       ).catch((errorMessage: string) => {
-        let errorToast: string = null;
+        let errorToast: string | null = null;
```

Comment:
Do we need to define the variable here? Or can we do just 
`let errorToast: string;`

Reply:
Will make the changes!